### ISSUE TITLE
Convert parameters to extension settings

### ISSUE THREAD

**Issue by alexdesignworks** [integratedexperts/behat-relativity#5]
Need to convert all parameters passed to the extension directly to be passed from `extension` section of `behat.yml`.

Supported parameters:
`breakpoints` - required at least one
`offset` - optional, defaults to `0`
`components` - optional

The structure of the settings should stay the same:
```
            # Supported screen breakpoints.
            breakpoints:
              mobile:
                width: 320
                height: 480
              tablet:
                width: 768
                height: 1024
              desktop:
                width: 992
                height: 1024
                default: true   <--- please note that we also have this and have to validate that only single breakpoint config has this key set.
              desktop_large:
                width: 1200
                height: 900
            # Vertical offset.
            offset: 60
            # List of test components.
            components:
              'page': "#page"         
```

**Comment by bladeaweb** [comment]
Fixed by #7

**Comment by alexdesignworks** [comment]
Released in `0.2.0`
